rise/static/main.js

```diff
@@ -988,26 +1001,33 @@ define([
       window.open(url, '_blank');
     }
 
-    actions.register({ help: 'open the nbconfigurator page for RISE',
-                       handler: nbconfigurator},
-                     "rise-nbconfigurator", "RISE");
+    actions.register(
+        {help: 'open the nbconfigurator page for RISE',
+         handler: nbconfigurator},
+        "rise-nbconfigurator", "RISE");
 
     // mostly for debug / information
-    actions.register({ help   : 'output RISE configuration in console, for debugging mostly',
-                       handler: showConfig},
-                     "rise-dump-config", "RISE");
+    actions.register(
+        {help   : 'output RISE configuration in console, for debugging mostly',
+         handler: showConfig},
+        "rise-dump-config", "RISE");
 
   }
 
+  // could maybe become configurable
+  let restoreTimeout = 500;
+  // when going too short, like 250, I can see selection behaving oddly
+  let autoSelectTimeout = 500;
+
```

Comment:
Let's keep it internal for now...

Reply:
I have a little problem with that; it turns out that the current release exhibits problems for me; in many cases, the selected code cell ends up displayed with a wrong size; mostly it's too high, and I have to pretend that I move up and down to get it right again, it's quite painful

investigating this issue, I haven't been able to go to the core of it, but I noticed that increasing this autoselecttimeout setting made things less awkward; not perfect, but much better.

for now I do it right in the code, but I cannot yet tell what is the best compromise, so I'd like to be able to change it quickly from the configurator dashboard

---
as to the restore timeout, I planned to add it together with autoselecttimeout because I was at it, but I don't care much if it's not tunable.

what do you think ?

Reply:
Go for it! Let's do it configurable if that helps you. I would be nice to know what is the root cause of the issue, but that's a task for later if the timing helps you enough well.